opennms-base-assembly/src/main/filtered/etc/events/opennms.events.xml

```diff
@@ -914,13 +914,15 @@
                         &lt;p&gt;This event is generated when node outage processing
                         determines that the critical path IP address/service for
                         this node is not responding..&lt;/p&gt;
+                        &lt;p&gt;More information on the affected node(s) can be found at
+                        the &lt;a href="opennms/pathOutage/index.jsp"&gt;Path Outages&lt;/a&gt; page.&lt;/p&gt;
                 </descr>
     <logmsg dest="logndisplay">
                         %parm[nodelabel]% path outage. Critical path =
                         %parm[criticalPathIp]% %parm[criticalPathServiceName]%
                 </logmsg>
     <severity>Major</severity>
-    <alarm-data reduction-key="%uei%:%dpname%:%nodeid%" alarm-type="3" auto-clean="false"/>
+    <alarm-data reduction-key="%uei%:%parm[criticalPathIp]%:%parm[criticalPathServiceName]%" alarm-type="3" auto-clean="false"/>
```

Comment:
The %dpname% token represents the "distributed poller" responsible for initiating the event. It is not part of the node, it represents the OpenNMS system responsible for generating the event so it still needs to be included in the reduction key.

This terminology is evolving as we finally implement our distributed monitoring concept (after 15 long years of incubation) with the work on Minion and on HA clustering for OpenNMS. We won't use the "distributed poller" term any more but if you see "monitoring system" in the UI or database, that refers to the same concept of the management system that detected the event.